include/ncengine/graphics/Camera.h

```diff
@@ -44,6 +51,9 @@ class Camera : public FreeComponent
          */
         auto ProjectionMatrix() const noexcept -> DirectX::FXMMATRIX { return m_projection; }
 
+        /** @brief Unproject 2D normalized device coordinates into 3D points on the near and far planes. */
+        auto UnprojectToNearFarPlanes(const Vector2& normalizedDeviceCoords) const -> NearFarPoints;
```

Comment:
I prefer "CastToNearFarPlanes", though I'm not too picky. It's a little shorter, and to me is easier to understand.

Reply:
Sure, updated.